eureka-webapp/src/main/webapp/assets/js/eureka.registration.js

```diff
@@ -137,7 +137,10 @@ window.eureka.registration = new function () {
 				},
 				error: function (xhr, err) {
 					$('#passwordChangeFailure').show();
-					$('#passwordErrorMessage').text(xhr.responseText)
+					$('#passwordErrorMessage').text(xhr.responseText);
+					$("#passwordErrorMessage:contains('login')").html(function(_, html) {
+                                            return html.replace(/(login)/g, '<a href="/login">$1</a>');
```

Comment:
I don't want to use the responseText at all. The javascript should include its own message.